main.py

```diff
@@ -78,7 +82,12 @@ def execute_all():
     print(f'Completed: {progress} out of {total_count}')
 
     # GeoCASe data
-    geocase_data()
+    geocase_specimens()
+    progress += 1
+    print(f'Completed: {progress} out of {total_count}')
```

Comment:
See if you can replace the print statements to logging functions. https://docs.python.org/3/library/logging.html You can check the enrichment service for this as well